lib/tasks/swaggerize.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+if defined?(RSpec)
```

Comment:
Do we just not want it to blow up if RSpec isn't installed, for example on a production only system?

Reply:
Uhh, when does this file get loaded?

I don't know, but chances are the answer is "more often than when you try to run that task". And if that's the case, we could accidentally load it in production as part of legitimate step and... BOOM

I prefer not adding task silently over blowing up :)

Reply:
I was asking if that was the reason, rather than suggesting it. Thought that it might be better to define that it was only loaded during test (and possibly development) explicitly, rather than just a magic dependency.